src/utils/style-formatter.js

```diff
@@ -38,16 +38,16 @@ const getColor = ({ useColorExpression, colorExpression, color }, defaultColor,
 };
 
 export default {
-  getStyles({ style, disabled, theme, element }) {
+  getStyles({ style = {}, disabled, theme, element }) {
     let styles = 'width: 100%;height: 100%;transition: transform .1s ease-in-out;position: absolute;';
-    const { font, background, border } = style;
+    const { font = {}, background = {}, border = {} } = style;
     const primaryColor = theme.getDataColorSpecials().primary;
     // enable
     styles += disabled ? formatProperty('opacity', 0.4) : formatProperty('cursor', 'pointer');
     // font
     styles += formatProperty('color', getColor(font, '#ffffff', theme));
-    font.style.bold && (styles += formatProperty('font-weight', 'bold'));
-    font.style.italic && (styles += formatProperty('font-style', 'italic'));
+    font.style && font.style.bold && (styles += formatProperty('font-weight', 'bold'));
```

Comment:
Seems to me like the default value for `bold` has changed here? Since font style will now be an empty object right? Was that intentional?

Reply:
You're right. nice find. I'm working on putting all defaults in a separate folder so style-formatter and ext always map correctly. I can ping you when I'm done so you don't have to review more than necessary

Reply:
You're right. nice find. I'm working on putting all defaults in a separate folder so style-formatter and ext always map correctly. I can ping you when I'm done so you don't have to review more than necessary

Reply:
Great as my only concern here would be that the default values have changed in the mashup case. Which I believe would be a breaking change, unless the API is experimental.

Reply:
fyi the api hasn't been published yet so we are good on that front. But it should correspond to creating a default button in the client